src/Akka.Streams.Kafka/Stages/CommittableConsumerStage.cs

```diff
@@ -107,78 +101,29 @@ public override void PreStart()
                     break;
             }
 
-            _messagesReceived = GetAsyncCallback<ConsumerRecord<K, V>>(MessagesReceived);
+            _messagesReceived = GetAsyncCallback<ConsumeResult<K, V>>(MessagesReceived);
             _partitionsAssigned = GetAsyncCallback<IEnumerable<TopicPartition>>(PartitionsAssigned);
-            _partitionsRevoked = GetAsyncCallback<IEnumerable<TopicPartition>>(PartitionsRevoked);
+            _partitionsRevoked = GetAsyncCallback<IEnumerable<TopicPartitionOffset>>(PartitionsRevoked);
             ScheduleRepeatedly(TimerKey, _settings.PollInterval);
         }
 
         public override void PostStop()
         {
-            _consumer.OnRecord -= HandleOnMessage;
-            _consumer.OnConsumeError -= HandleConsumeError;
-            _consumer.OnError -= HandleOnError;
-            _consumer.OnPartitionsAssigned -= HandleOnPartitionsAssigned;
-            _consumer.OnPartitionsRevoked -= HandleOnPartitionsRevoked;
-
             Log.Debug($"Consumer stopped: {_consumer.Name}");
             _consumer.Dispose();
 
             base.PostStop();
         }
+        
+        protected override void OnTimer(object timerKey) => PullQueue();
```

Comment:
So I know you're probably planning on changing this, but we don't actually need `OnTimer` or any sort of timer-driven behavior anymore now that we're just pulling from the Kafka topic on-demand, right?

Reply:
Right

Reply:
Good, just wanted to understand